src/data/community-events.json

```
@@ -35,6 +35,15 @@
     "startDate": "2023-03-10",
     "endDate": "2023-03-29"
   },
+  {
+    "title": "ETH Beijing",
+    "to": "https://ethbeijing.com/",
```

```suggestion
    "to": "https://ethbeijing.xyz/",
```

Updated based on comment